.github/workflows/build-and-deploy.yml

```diff
@@ -89,3 +89,61 @@ jobs:
         healthcheck: ${{ env.HEALTHCHECK_CMD }}
         db-seed: false
         smoke-test: false
+
+  deploy:
+    name: Deploy environments
+    if: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' }}
+    environment:
+      name: ${{ matrix.environment }}
+      url: ${{ steps.deploy_app.outputs.environment_url }}
+    needs: [build]
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      pull-requests: write
+    strategy:
+      max-parallel: 1
+      matrix:
+        environment: [staging, production]
+
+    steps:
+      - name: Deploy app to ${{ matrix.environment }}
+        id: deploy_app
+        uses: DFE-Digital/github-actions/deploy-to-aks@master
+        with:
+          azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          azure-subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          environment: ${{ matrix.environment }}
+          sha: ${{ needs.build.outputs.docker-image-tag }}
+          terraform-base: ${{ env.TERRAFORM_BASE }}
+          healthcheck: ${{ env.HEALTHCHECK_CMD }}
+          slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+          db-seed: ${{ matrix.environment == 'staging' && 'true' || 'false'}}
```

Comment:
we don't normally seed staging, but if that's a requirement you'll have to update the Makefile as it's written for review apps